mcc/GetSetAttrs.c

```diff
@@ -598,6 +627,52 @@ IPTR mSet(struct IClass *cl, Object *obj, struct opSet *msg)
       }
       break;
 
+      case MUIA_TextEditor_HorizontalSlider:
+      {
+        if(ti_Data)
+	{
+          if(data->shown == FALSE)
+          {
+            data->hslider = (void *)ti_Data;
+
+            // disable the hslider right away if the texteditor
+            // gadget is disabled as well.
+            if(isFlagSet(data->flags, FLG_Ghosted))
+              set(data->hslider, MUIA_Disabled, TRUE);
+
+            // Set the gadget to be always scrollable when there is a H.scroller
+            setFlag(data->flags, FLG_HScroll);
+
+            DoMethod(data->hslider, MUIM_Notify,
+                MUIA_Prop_Release, MUIV_EveryTime,
+                obj, 3, MUIM_NoNotifySet, MUIA_TextEditor_Prop_Release, MUIV_TriggerValue);
+            DoMethod(data->hslider, MUIM_Notify,
+                MUIA_Prop_First, MUIV_EveryTime,
+                obj, 3, MUIM_NoNotifySet, MUIA_TextEditor_HSlider_Pos, MUIV_TriggerValue);
+            DoMethod(obj, MUIM_Notify,
+                MUIA_TextEditor_HSlider_Pos, MUIV_EveryTime,
+                data->hslider, 3, MUIM_NoNotifySet, MUIA_Prop_First, MUIV_TriggerValue);
+            DoMethod(obj, MUIM_Notify,
+                MUIA_TextEditor_HSlider_Ent, MUIV_EveryTime,
+                data->hslider, 3, MUIM_NoNotifySet, MUIA_Prop_Entries, MUIV_TriggerValue);
+            DoMethod(obj, MUIM_Notify,
+                MUIA_TextEditor_HSlider_Vis, MUIV_EveryTime,
+                data->hslider, 3, MUIM_NoNotifySet, MUIA_Prop_Visible, MUIV_TriggerValue);
+            DoMethod(obj, MUIM_Notify,
+                MUIA_TextEditor_Prop_DeltaFactor, MUIV_EveryTime,
+                data->hslider, 3, MUIM_NoNotifySet, MUIA_Prop_DeltaFactor, MUIV_TriggerValue);
+          }
+        }
+	else
```

Comment:
else block has incorrect indentation. add space to move else block one space forward.